caches/partyline/src/main/java/org/commonjava/maven/galley/cache/partyline/PartyLineCacheProviderConfig.java

```diff
@@ -31,9 +31,10 @@
 
     private Integer defaultTimeoutSeconds;
 
-    public PartyLineCacheProviderConfig( final File cacheBasedir )
+    public PartyLineCacheProviderConfig( final File cacheBasedir, final Boolean timeoutProcessing )
```

Comment:
There is a method 'withTimeoutProcessingEnabled'. maybe just update Indy side to call it?

Reply:
@ruhan1 Indy uses PartyLineCacheProviderFactory to build, I update to use this when creating partyline CacheProvider instead